README.md

```diff
@@ -45,35 +46,36 @@ Please do not open the .pem file as this might corrupt the key and you will need
 
 ## Settings for new registrations
 
-`Only allow existing Wordpress users to link their Yoti account` - This setting allows a new user to Register and Log in by using their Yoti. A new user who registeres this way will be set to the `Subscriber` role in WordPress. If enabled, when a new user tries to scan the Yoti QR code, they will be redirected back to the login page with an error message displayed.
+`Only allow existing WordPress users to link their Yoti account` - This setting allows a new user to Register and Log in by using their Yoti. A new user who registeres this way will be set to the `Subscriber` role in WordPress. If enabled, when a new user tries to scan the Yoti QR code, they will be redirected back to the login page with an error message displayed.
 
-`Attempt to link Yoti email address with Wordpress account for first time users` - This setting enables linking a Yoti account to a WordPress user if the email from both platforms is identical.
+`Attempt to link Yoti email address with WordPress account for first time users` - This setting enables linking a Yoti account to a WordPress user if the email from both platforms is identical.
 
 ## Docker
 
-We provide a WordPress Docker container that includes the Yoti plugin.
+We provide a WordPress [Docker](https://docs.docker.com/) container that includes the Yoti plugin.
 
 ### Setup
 
-Clone this repository and go into the folder:
+Clone this repository and go into the folder `yoti-wordpress`:
 
 ```shell
-cd yoti-wordpress
+$ git clone https://github.com/getyoti/yoti-wordpress.git
+$ cd yoti-wordpress
 ```
 
 Rebuild the images if you have modified the `docker-compose.yml` file:
 
 ```shell
-docker-compose build --no-cache
+$ docker-compose build --no-cache
 ```
 
 Build the containers:
 
 ```shell
-docker-compose up -d
+$ docker-compose up -d
 ```
 
-After the command has finished running, go to [http://localhost:7000](http://localhost:7000) and follow the instructions.
+After the command has finished running, go to [https://localhost:7001](https://localhost:7001) and follow the instructions.
```

Comment:
Again, why do we use a different port for every SDK?

Reply:
We use different port to avoid conflict when running the containers simultaneously.

Reply:
ok